app/destination/page.js

```diff
@@ -74,41 +105,56 @@ export const Destinations = () => {
         </section>
         <section className="card">
           <h2>Possible destinations</h2>
-          {/* TASK - React 1 week 2 */}
-          {/* Add all 4 planets! Europa, Moon, Mars, Titan  */}
-          {/* Use the README.md file for descriptions */}
-          {/* Create a <PlanetCard /> component, which accepts the following properties: */}
-          {/* name, description, thumbnail, isSelected, onAddOrRemovePlanet */}
-          <div className={styles.planetCard}>
-            <img className={styles.planetThumbnail} src="/destination/image-europa.png" alt="" />
-            <div className={styles.planetDescription}>
-              <h2>EUROPA {isPlanetSelected ? "- SELECTED" : ""}</h2>
-              <p>Lorem ipsum...</p>
-            </div>
-            <button 
-              className="roundButton" 
-              onClick={() => onAddOrRemovePlanet('Pluto', 0)}
-            > 
-              {isPlanetSelected ? "REMOVE" : "ADD PLANET"}
-            </button>
-          </div>
-          <div className={styles.planetCard}>
-            <img className={styles.planetThumbnail} src="/destination/image-europa.png" alt="" />
-            <div className={styles.planetDescription}>
-              <h2>EUROPA {isPlanetSelected ? "- SELECTED" : ""}</h2>
-              <p>Lorem ipsum...</p>
-            </div>
-            <button 
-              className="roundButton" 
-              onClick={() => onAddOrRemovePlanet('Pluto', 0)}
-            > 
-              {isPlanetSelected ? "REMOVE" : "ADD PLANET"}
-            </button>
-          </div>
+
+          {planetsArray.map((planet, index) => (
+            <PlanetCard
+              key={planet.id}
+              name={planet.name}
+              description={planet.description}
+              thumbnail={planet.thumbnail}
+              isSelected={selectedPlanets.includes(
+                planet.name
+              )}
+              onAddOrRemovePlanet={onAddOrRemovePlanet}
+            />
+          ))}
         </section>
       </main>
     </div>
   );
+};
+
+function PlanetCard({
+  name,
+  index,
+  description,
+  thumbnail,
+  isSelected,
+  onAddOrRemovePlanet,
+}) {
+  return (
+    <>
+      <div className={styles.planetCard}>
+        <img
+          className={styles.planetThumbnail}
+          src={thumbnail}
+          alt={name}
```

Comment:
This `alt` is not providing extra information, so it can be marked as a decorative image `alt=""`.
This is mostly for the screenreaders, and people who are non-visual users (e.g. blind people) who are using the screenreaders to browse the web